Dockerfile

```diff
@@ -71,3 +71,15 @@ RUN mkdir /opt/cmake-tmp && \
 
 # Make the SDK universally readable
 RUN chmod -R a+rX ${ANDROID_HOME}
+
+#### Install Testing Realm Object Server
+# Add realm repo
+RUN apt-get update -qq \
```

Comment:
I don't think that's a good idea. ROS should have its own Dockerfile and run its own container.